packages/owl-bot/test/commit-post-processor-update.ts

```diff
@@ -177,21 +179,48 @@ describe('commitPostProcessorUpdate', () => {
     assert.match(log, /Updates from OwlBot/);
   });
 
-  it('adds commit when .OwlBot.yaml contains no flag', async () => {
+  function makeRepoWithPendingCommit(): {origin: string; clone: string} {
     const origin = makeOrigin();
     const clone = cloneRepo(origin);
     makeDirTree(clone, [`${yamlPath}:deep-remove-regex:\n  - /pasta.txt`]);
     cmd(`git add ${yamlPath}`, {cwd: clone});
     const copyTag = copyTagFrom(yamlPath, 'abc123');
     cmd(`git commit -m "Copy-Tag: ${copyTag}"`, {cwd: clone});
     makeDirTree(clone, ['a.txt:The post processor ran.']);
+    return {origin, clone};
+  }
+
+  it('adds commit when .OwlBot.yaml contains no flag', async () => {
+    const {origin, clone} = makeRepoWithPendingCommit();
     await commitPostProcessorUpdate(prepareArgs(clone));
     const log = cmd('git log --format=%B main', {cwd: origin}).toString(
       'utf-8'
     );
     assert.match(log, /Updates from OwlBot/);
   });
 
+  it('updates pr title and body', async () => {
+    const {origin, clone} = makeRepoWithPendingCommit();
+    const args = prepareArgs(clone);
+    args['new-pull-request-text-path'] = tmp.fileSync().name;
+    fs.writeFileSync(
+      args['new-pull-request-text-path'],
+      'updated title\n\nUpdated body.'
+    );
+    // commitPostProcessorUpdate() should issue a PATCH request to update the PR.
+    nock('https://api.github.com')
```

Comment:
You can capture the mocked call and then assert that it was called.

```
const scope = nock('...').patch(...);
...


scope.done(); // will throw if not all the calls were made.
```

Otherwise, this test could pass if the update mock was not called.

Reply:
Done.